src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbStreamsBySharedTable.java

```diff
@@ -104,117 +106,104 @@ protected MtGetRecordsResult getAllRecords(GetRecordsRequest request, StreamArn
     @Override
     protected MtGetRecordsResult getRecords(GetRecordsRequest request, MtStreamArn mtStreamArn) {
```

Comment:
This method changed a bit more than expected, because adding the additional properties to the mt result allowed me to simplify it a lot.